### PR TITLE
Adds the ability to hide the completed AR UI by tapping on the screen

### PR DESCRIPTION
**Pull request by orta**
Fixes DISCO-552

Adds support for toggling the UI, to make it easier to take screenshots when using the AR mode.

### CODE REVIEW

File: Artsy/View_Controllers/ARVIR/VCs/ARAugmentedFloorBasedVIRViewController.m [hunk 1]
```diff
@@ -425,6 +428,14 @@ - (void)resetAR
 - (IBAction)screenTapped:(UITapGestureRecognizer *)gesture
 {
     [self.interactionController tappedOnScreen:gesture];
+
+    if ([self.informationView isAtLastState]) {
+        [UIView animateWithDuration:ARAnimationQuickDuration animations:^{
+            self.betaImageView.alpha = !self.betaImageView.alpha;
+            self.backButton.alpha = !self.backButton.alpha;
+            self.resetARButton.alpha = !self.resetARButton.alpha;
```
Comment (ashfurrow) [inline Artsy/View_Controllers/ARVIR/VCs/ARAugmentedFloorBasedVIRViewController.m]: Any reason to make this implementation reliant on the quirks of C bool/float interpolation? If it were a little easier to parse for non-ObjC devs, we could get more contributors 😄

Comment (orta) [reply Artsy/View_Controllers/ARVIR/VCs/ARAugmentedFloorBasedVIRViewController.m]: It looks messier now with ternaries, but sure - I'm alright with that 👍

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Needs testing on a Phone if change is non-trivial</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_travis">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by orta** [comment]
Assuming the only feedback meant everything else was 👍

**Pull request merged**